source/javascripts/components/ProductTour/tips.ts

```diff
@@ -0,0 +1,47 @@
+import { Tips } from "./types";
+
+export const tips: Tips[] = [
+	{
+		id: "workflows",
+		title: "Welcome to the workflow editor",
+		description:
+			"Here you can build workflows using our library of steps and automate tasks to replace repetitive, manual processes."
+	},
+	{
+		id: "code-signing",
+		title: "Code signing",
+		description:
+			"Here you can upload sensitive files to Bitrise to allow you to distribute your app." +
+			"Find out how to set up code signing for your project type.",
+		link: "https://devcenter.bitrise.io/en/code-signing.html"
+	},
+	{
+		id: "secrets",
+		title: "Secrets",
+		description: "Find out how to set up secrets, how to edit them, expose them to pull requests and more.",
+		link: "https://devcenter.bitrise.io/en/builds/secrets.html"
+	},
+	{
+		id: "env-vars",
+		title: "Env Vars",
+		description:
+			"Here you the define environment variables that the steps need to access in a build. " +
```

Comment:
Typo in: `Here you the define environment`

Reply:
I used the copy from the ticket and asked Hunter if this is what he wanted and the response is: 

"It was actually supposed to be as you can see in the screenshot, i.e. this is where you do this rather than this is where you can do it. I guess it depends on how often env vars are needed for a build to run successfully. If it's something you might or might not do, 'can' would work better. If it's something you will (almost) always do, it's better without 'can'."

So I'll leave the copy as is as this is what was intended.